src/components/HAccordion.svelte

```diff
@@ -0,0 +1,59 @@
+<script lang="ts">
+  import { createEventDispatcher, type Snippet } from 'svelte';
+  import { Button } from '@frequency-chain/style-guide';
+  import Vector from '../lib/assets/Vector.svg';
+
+  const dispatch = createEventDispatcher();
+
+  interface HAccordionProps {
+    state: 'open' | 'closed';
```

Comment:
Is there anything in the way of us using `isOpen` and using a boolean so we don't have to do so much equals logic?